app/src/main/java/uk/ac/ebi/atlas/experimentimport/admin/SingleCellExperimentAdminController.java

```diff
@@ -27,4 +35,13 @@ public SingleCellExperimentAdminController(DataFileHub dataFileHub,
     public String fooBar() {
         return "";
     }
+
+    @GetMapping(value = "/{experimentAccession}")
+    public void updateSingleCellExperiment(
+            @PathVariable String experimentAccession,
+            @RequestParam(value = "isPrivate") Boolean isPrivate,
+            ScxaExperimentCrud scxaExperimentCrud) {
+        UUID accessKeyUUID = scxaExperimentCrud.createExperiment(experimentAccession, isPrivate);
```

Comment:
`createExperiment ` method,  serve these things:

1. Clear the existing cache and allows the application to cache the latest data sets
2. Reads experiment from the DB by experiment accession
3. Creates a new experiment in the DB if the access key is empty 
4.  Updates existing experiment if the access key is available

what could be the reason to call this method here? only to clear cache? or only update an experiment or both?

Reply:
We would like to provide this URL to update or add new experiments to the cache.